contributors/devel/controllers.md

```diff
@@ -22,84 +22,76 @@ When you're writing controllers, there are few guidelines that will help make su
 you're looking for.
 
 1. Operate on one item at a time.  If you use a `workqueue.Interface`, you'll be able to queue changes for a
- particular resource and later pop them in multiple “worker” gofuncs with a guarantee that no two gofuncs will
- work on the same item at the same time.
-
- Many controllers must trigger off multiple resources (I need to "check X if Y changes"), but nearly all controllers
- can collapse those into a queue of “check this X” based on relationships.  For instance, a ReplicaSetController needs
- to react to a pod being deleted, but it does that by finding the related ReplicaSets and queuing those.
+particular resource and later pop them in multiple “worker” gofuncs with a guarantee that no two gofuncs will
+work on the same item at the same time.
 
+   Many controllers must trigger off multiple resources (I need to "check X if Y changes"), but nearly all controllers
+can collapse those into a queue of “check this X” based on relationships.  For instance, a ReplicaSetController needs
```

Comment:
Can you also tighten the column width to something smaller? I am not sure if there is any ideal fixed size but the current is too long.